walk.go

```diff
@@ -360,3 +366,38 @@ func signatureEntries(root string) []Entry {
 
 	return sigEntries
 }
+
+// keywordEntries returns a slice of entries that ensure that a manifest
+// generated with a particular keyword set will still be recognised as having
+// that keyword set. Namely this is [/set <keywords>, /set <none>].
+func keywordEntries(keywords []string) []Entry {
+	// Convert all of the keywords to zero-value keyvals.
+	kvs := []string{}
+	for _, kw := range keywords {
+		kvs = append(kvs, kw+"=")
+	}
+
+	// Create a /set <kvs> and an /unset.
+	return []Entry{
+		{
+			Type: CommentType,
+			Raw:  fmt.Sprintf("#%16s%s", "keywords: ", strings.Join(keywords, ",")),
```

Comment:
If we do go this route, perhaps this comment could instead be like "# here are the set of keywords used in this manifest"